Dockerfile-prod

```diff
@@ -0,0 +1,27 @@
+FROM quay.io/sameersbn/ubuntu:latest
+MAINTAINER Alexander Trost <galexrt@googlemail.com>
+
+ENV ZULIP_VERSION="1.3.10" DATA_DIR="/data"
```

Comment:
I would just change the `ZULIP_VERSION` to `latest` so it always downloads and installs the latest version or the user specified.